stacks/empty/devfile.yaml

```diff
@@ -0,0 +1,7 @@
+schemaVersion: 2.2.0
+metadata:
+  name: empty
```

Comment:
might be better to name it as scratch to use the common analogy in the container world with https://hub.docker.com/_/scratch